mypy/binder.py

```diff
@@ -165,21 +175,20 @@ def pop_frame(self, fall_through: int = 0) -> Frame:
 
         return result
 
-    def get_declaration(self, node: Node) -> Type:
-        if isinstance(node, (RefExpr, SymbolTableNode)) and isinstance(node.node, Var):
-            type = node.node.type
+    def get_declaration(self, expr: BLval) -> Type:
+        if isinstance(expr, (RefExpr, SymbolTableNode)) and isinstance(expr.node, Var):
+            type = expr.node.type
```

Comment:
The test here is never true, and I think the original (changed in one of my previous PRs) was never true either. I wonder why.